lang/js/src/main/scala/com/wavesplatform/lang/WavesContext.scala

```diff
@@ -6,7 +6,7 @@ object WavesContext {
   val env = new Environment {
     override def height: Int                                         = impl.Environment.height
     override def networkByte: Byte                                   = impl.Environment.networkByte
-    override def transaction: Tx                                     = impl.Environment.transaction
+    override def inputEntity: Tx                                     = impl.Environment.transaction
```

Comment:
shouldn't it be `T :+: Order :+: CNil` ?

Reply:
Fixed